Refresher/Patching/EbootPatcher.cs

```diff
@@ -332,6 +333,21 @@ public List<Message> Verify(string url, bool patchDigest)
             messages.Add(new Message(MessageLevel.Warning,
                                      "Could not find the digest in the EBOOT. Resulting EBOOT may still work depending on game."));
 
+        if (this.GenerateRpcs3Patch)
+        {
+            if (string.IsNullOrWhiteSpace(this.GameVersion))
+            {
+                messages.Add(new Message(MessageLevel.Error,
+                    "Missing game version!")); 
+            }
+
+            if (string.IsNullOrWhiteSpace(this.PpuHash))
+            {
+                messages.Add(new Message(MessageLevel.Error,
+                    "Missing PPU hash!")); 
```

Comment:
These types of errors aren't really helpful. You could explain why they're necessary, how to find them, etc.